src/SignalR/clients/csharp/Client.Core/src/IRetryPolicy.cs

```diff
@@ -0,0 +1,12 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+
+namespace Microsoft.AspNetCore.SignalR.Client
+{
+    public interface IRetryPolicy
```

Comment:
You actually call this `IReconnectPolicy` in the TypeScript

https://github.com/aspnet/AspNetCore/blob/a6fd8da9e87d29515db3196cae7b70ea40f90381/src/SignalR/clients/ts/signalr/src/IReconnectPolicy.ts#L5

Reply:
I was going to bring up the idea of renaming this in the TS client too for Preview6.

The reason being is we might want to use the same interface to configure retry behavior after a failed start. Currently, we never retry on initial start failures no matter the reconnect policy, and we don't have any immediate plans to start doing so. But if the future we want to add a retry-start policy, it would be nice if the interface it used was called IRetryPolicy instead of IReconnectPolicy.

Thoughts?

Reply:
I could be convinced either way as far as the naming goes. As long as we are consistent. 

> Currently, we never retry on initial start failures

Don't we try to fallback? Would you configure retry policies that would take precedence over transport fall back? The next question becomes would we go through the retry strategy over all the transports? Questions for another time though.